From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Thu, 24 Aug 2023 22:33:56 +0900
Subject: [PATCH] default song color to random

---
 public/src/js/songselect.js | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/public/src/js/songselect.js b/public/src/js/songselect.js
index 705a0d0..9e72848 100644
--- a/public/src/js/songselect.js
+++ b/public/src/js/songselect.js
@@ -17,6 +17,7 @@ class SongSelect{
 			this.canvas.style.imageRendering = "pixelated"
 		}
 
+		const defaultColor = Math.floor(Math.random() * 0xFFFFFF).toString(16).padStart(6, "0");
 		this.songSkin = {
 			"selected": {
 				background: "#ffdb2c",
@@ -72,10 +73,10 @@ class SongSelect{
 			},
 			"default": {
 				sort: null,
-				background: "#ececec",
-				border: ["#fbfbfb", "#8b8b8b"],
-				outline: "#656565",
-				infoFill: "#656565"
+				background: `#${defaultColor}`,
+				border: [`#${defaultColor}`, `#${defaultColor}`],
+				outline: `#000000`,
+				infoFill: `#${defaultColor}`
 			}
 		}
 		
-- 
2.34.1

