From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: yuuki <>
Date: Thu, 5 Oct 2023 15:26:46 +0000
Subject: [PATCH] fix docker exec

---
 Dockerfile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Dockerfile b/Dockerfile
index f8494db..fa5e608 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -3,4 +3,5 @@ COPY . /app
 WORKDIR /app
 RUN find wheels -name '*.whl' -print0 | xargs -0 pip install
 RUN sed -i 's/\r$//' wait-for-it.sh
+RUN chmod +x wait-for-it.sh
 CMD ["./wait-for-it.sh","-t","300","mongo:27017","--","python","app.py","34801","-b","0.0.0.0"]
-- 
2.39.2

