From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: yuuki <>
Date: Thu, 5 Oct 2023 15:27:05 +0000
Subject: [PATCH] visible transparent song panel

---
 public/src/js/canvasdraw.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/public/src/js/canvasdraw.js b/public/src/js/canvasdraw.js
index 8f4e296..e829b8e 100644
--- a/public/src/js/canvasdraw.js
+++ b/public/src/js/canvasdraw.js
@@ -153,7 +153,7 @@
 				y: 5,
 				force: !noForce
 			})
-			ctx.fillStyle = "transparent"
+			ctx.fillStyle = "rgb(0,0,0,.3)"
 			ctx.fillRect(0, 0, w, h)
 		}
 		if(config.cached){
-- 
2.39.2

