From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: yuuki <>
Date: Thu, 5 Oct 2023 15:44:31 +0000
Subject: [PATCH] send manifest json from Flask

---
 app.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/app.py b/app.py
index 76ac0c9..2090b9c 100644
--- a/app.py
+++ b/app.py
@@ -15,7 +15,7 @@ import os
 import time
 
 from functools import wraps
-from flask import Flask, g, jsonify, render_template, request, abort, redirect, session, flash, make_response, send_from_directory
+from flask import Flask, g, jsonify, render_template, request, abort, redirect, session, flash, make_response, send_from_directory, send_file
 from flask_caching import Cache
 from flask_session import Session
 from flask_wtf.csrf import CSRFProtect, generate_csrf, CSRFError
@@ -751,5 +751,9 @@ if __name__ == '__main__':
     def send_songs(path):
         return send_from_directory('public/songs', path)
 
+    @app.route(basedir + 'manifest.json')
+    def send_manifest_json():
+        return send_file('public/manifest.json')
+
     app.run(host=args.bind_address, port=args.port, debug=args.debug)
 
-- 
2.39.2

